core/src/sql/statements/define/user.rs

```diff
@@ -251,37 +251,16 @@ impl Display for DefineUserStatement {
 
 impl InfoStructure for DefineUserStatement {
 	fn structure(self) -> Value {
-		let Self {
-			name,
-			base,
-			hash,
-			roles,
-			duration,
-			comment,
-			..
-		} = self;
-		let mut acc = Object::default();
-
-		acc.insert("name".to_string(), name.structure());
-
-		acc.insert("base".to_string(), base.structure());
-
-		acc.insert("passhash".to_string(), hash.into());
-
-		acc.insert(
-			"roles".to_string(),
-			Value::Array(roles.into_iter().map(|r| r.structure()).collect()),
-		);
-
-		let mut dur = Object::default();
-		dur.insert("token".to_string(), duration.token.into());
-		dur.insert("session".to_string(), duration.session.into());
-		acc.insert("duration".to_string(), dur.to_string().into());
-
-		if let Some(comment) = comment {
-			acc.insert("comment".to_string(), comment.into());
-		}
-
-		Value::Object(acc)
+		Value::from(map! {
+			"name".to_string() => self.name.structure(),
+			"base".to_string() => self.base.structure(),
+			"hash".to_string() => self.hash.into(),
+			"roles".to_string() => self.roles.into_iter().map(Ident::structure).collect(),
+			"duration".to_string() => Value::from(map! {
```

Comment:
This changed, is that intensional? If so we need to update it in Surrealist

Reply:
Yes `passhash` has been renamed to `hash`. The rest should be the same?

Reply:
CC @macjuul for `passhash` -> `hash` rename